signalflow/channel_test.go

```diff
@@ -0,0 +1 @@
+package signalflow
```

Comment:
Nothing to add here, I guess? (Why keep the file?)

Reply:
Added a test.